README.md

```diff
@@ -84,8 +84,9 @@ On all three:
 * `autoreconfigure::branch`
 * If you are forking into a repo and using a different branch name, update the autoreconfigure command to point to the right branch.
 * `ssh_keys`
- * Configure as many license keys as you want for administrators to log in.
- * Make sure there is at least one key for jenkins-slave on the repo matching the `jenkins::private_key` provisioned on the master.
+ * Configure as many public ssh-keys as you want for administrators to log in. It's recommended at least one for root.
+ * On the repo machine make sure there is at least one key for the jenkins-slave user matching the
+  `jenkins::private_key` provisioned on the master.
```

Comment:
This is still a bit fuzzy. It doesn't clarify what are public and what are private keys. I would suggest to include that as or similar to the phrase in the referenced issue.